dev/source/docs/building-the-code.rst

```diff
@@ -15,18 +15,36 @@ Setting up the Build Environment
 - :ref:`Setup the Build Environment on Windows <building-setup-windows>`
 - :ref:`Setup the Build Environment on MacOSX <building-setup-mac>`
 
+
 Building / Compiling
 --------------------
 
-**Linux / MacOSX users:**
+Once the build environment is set up as above, run the following commmands prior to the initial build. (For Windows users, this is under WSL, as set up above):
 
-- Linux and MacOSX users should build with waf as described in `BUILD.md <https://github.com/ArduPilot/ardupilot/blob/master/BUILD.md>`__.
+- `git submodule init`
+- `git submodule update --recursive`
+- `./Tools/gittools/submodule-sync.sh`
+- `python3 waf configure --board *BoardName*`
```

Comment:
```suggestion
- `./waf configure --board *BoardName*`
```

Reply:
Hi! It seems for me that `./waf` doesn't work. It is depending on how your system python is configured. Calling `python3` explicitly is more universal. (And works when I test it)